rdf-star-council-report.bs

```diff
@@ -168,6 +169,23 @@ Not everyone in the W3C community is aware that the Process gives a precise defi
 We also encourage the objector and the WG chairs to work together to find a mutually agreeable
 schedule to accomplish the group's chartered work.
 
+<p id="rec-adopted-draft">Finally, in reviewing how [section 6.3.11.4 of the W3C
+Process](https://www.w3.org/policies/process/#revised-rec-features) applies to this charter, we
+noticed that the charter's use of ["Adopted
+Draft"](https://www.w3.org/policies/process/#adopted-draft) can be read to imply that the WG intends
+to continue work on the RDF 1.1 Recommendations by renaming them to "RDF 1.2". [Section
+6.3.11.4](https://www.w3.org/policies/process/#revised-rec-features) would prohibit doing this,
+since the RDF 1.1 Recommendations are not identified as [allowing new
+features](https://www.w3.org/policies/process/#allow-new-features). The [Process' definition of
+"Adopted Draft"](https://www.w3.org/policies/process/#adopted-draft) can also be read to apply to
+documents that are merely the basis of work on *new* technical reports, as it was intended here, so
```

Comment:
I don't think this is true. The only use of “Adopted Draft” in accordance with the Process is for existing technical reports. There is no definition or use of the term with regards to new technical reports: the entire list in which the term appears is prefaced as applying to continues work on a technical published under a different charter.

You could maybe argue that the charter template has its own use of “Adopted Draft” which is not the same as the Process's use of “Adopted Draft”... but I don't think you can reasonably argue that the Process has any other notion of it.

Reply:
I'm not entirely convinced, but the Process does _imply_ the meaning you're saying, so maybe we can talk about that implication instead of saying it's ambiguous. How's the [new wording](https://pr-preview.s3.amazonaws.com/jyasskin/draft-council-reports/pull/9.html#rec-adopted-draft)?

Reply:
Perfect. Approved. :)